certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -614,7 +622,7 @@
                             },
                             "CustomerShipAttn": {
                               "type": "string",
-                              "description": "CustomerShipAttn"
+                              "description": "The attention field indicating the individual being shipped to%2C if applicable."
```

Comment:
@PavloNikulinEnavate Can you please let us know the meaning of "..shipped to%2C"